registry.k8s.io/images/charts/images.yaml

```diff
@@ -4,3 +4,6 @@
     "sha256:56acd0d3805fee0ec8f071e0dfa41b926d4e5b6f5cf60436ffa42300f6dc17d5": ["v0.10.1"]
     "sha256:bc238183a94983d85483dcb028f2c6dbffe5dee8d9f2b46988680049eb708df9": ["v0.9.4"]
     "sha256:577b280e31a978a82d82da097c236b8ec4ae8323f1c3bfcc06d25eed45d0d874": ["v0.9.3"]
+- name: jobset
+  dmap:
+    "sha256:759f017466b06b525b0273474e2b70243114819b89e237900ce3219c2a081fdf": ["v0.8.0"]
```

Comment:
```shell
$ helm template oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/jobset --version v0.8.0 | grep registry.k8s.io/jobset/jobset:v0.8.0
Pulled: us-central1-docker.pkg.dev/k8s-staging-images/charts/jobset:v0.8.0
Digest: sha256:759f017466b06b525b0273474e2b70243114819b89e237900ce3219c2a081fdf
        image: registry.k8s.io/registry.k8s.io/jobset/jobset:v0.8.0
```

Reply:
@kannon92 @ahg-g btw, is this `registry.k8s.io/registry.k8s.io/jobset/jobset:v0.8.0` correct?

Reply:
I think you failed to construct the image endpoint.

Reply:
/hold for the above weird image endpoint.

Reply:
Thanks for that! I couldn't figure out how to view that.

That is not correct. It should be registry.k8s.io/jobset:v0.8.0.

Reply:
I guess that the generating release artifact process is not correct in the JobSet Makefil or any kind of hack scripts.

Reply:
Yea its correct for the manifests..

Reply:
Ah its probably the `helm chart push`.

Reply:
this is resolved.